scripts/lsstinstall

```diff
@@ -209,6 +209,8 @@ else
         eval "$__conda_setup" || fail "Unable to start conda"
     fi
     for profile in "$conda_path/etc/profile.d"/*.sh; do
+        # shellcheck source=/dev/null
+        [ "$(basename "$profile")" = "mamba.sh" ] && continue
```

Comment:
So the default conda profile.d directory has a mamba shell script in it that they don't want anyone to source and so we have to skip it?